python/rmm/_lib/memory_resource.pyx

```diff
@@ -628,6 +628,13 @@ cpdef _flush_logs():
 def enable_logging(log_file_name=None):
     """
     Enable logging of run-time events.
+
+    log_file_name:  str, optional
+        Name of the log file. If not specified, the environment variable
+        RMM_LOG_FILE is used. A TypeError is thrown if neither is available.
```

Comment:
This is minor, but is `TypeError` the best choice? I'm wondering if maybe `ValueError` is technically more correct, or even `NameError`, `KeyError` (since the env var wasn't in the dictionary), or `RuntimeError`. Just a suggestion, I don't have a strong opinion.

Reply:
Agreed -- `ValueError` sounds more appropriate here.